static/app/components/issues/groupListHeader.tsx

```diff
@@ -4,25 +4,37 @@ import {PanelHeader} from 'sentry/components/panels';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 
+import {Column} from './groupList';
+
 type Props = {
   withChart: boolean;
   narrowGroups?: boolean;
   showLastTriggered?: boolean;
+  withColumns?: Column[];
 };
 
 function GroupListHeader({
   withChart = true,
   narrowGroups = false,
   showLastTriggered = false,
+  withColumns = ['graph', 'event', 'users', 'assignee', 'lastTriggered'],
```

Comment:
Alternatively, so this is less clunky we can just do
`if (!withColumns) -> assume all columns are selected`